manifests/server/conf.pp

```diff
@@ -198,67 +241,71 @@
-  Optional[Boolean]                $rhostsrsaauthentication         = $ssh::server::params::rhostsrsaauthentication,
-  Boolean                          $strictmodes                     = true,
-  String                           $subsystem                       = 'sftp /usr/libexec/openssh/sftp-server',
-  Ssh::Syslogfacility              $syslogfacility                  = 'AUTHPRIV',
-  Boolean                          $tcpwrappers                     = simplib::lookup('simp_options::tcpwrappers', { 'default_value' => false }),
-  Variant[Boolean,Enum['sandbox']] $useprivilegeseparation          = $ssh::server::params::useprivilegeseparation,
-  Boolean                          $x11forwarding                   = false,
+  Array[String]                                           $acceptenv                       = $ssh::server::params::acceptenv,
+  Optional[Array[String]]                                 $allowgroups                     = undef,
+  Optional[Array[String]]                                 $allowusers                      = undef,
+  String                                                  $authorizedkeysfile              = '/etc/ssh/local_keys/%u',
+  Optional[Stdlib::Absolutepath]                          $authorizedkeyscommand           = undef,
+  String                                                  $authorizedkeyscommanduser       = 'nobody',
+  Stdlib::Absolutepath                                    $banner                          = '/etc/issue.net',
+  Boolean                                                 $challengeresponseauthentication = false,
+  Optional[Array[String]]                                 $ciphers                         = undef,
+  Integer                                                 $clientalivecountmax             = 0,
+  Integer                                                 $clientaliveinterval             = 600,
+  Variant[Boolean,Enum['delayed']]                        $compression                     = 'delayed',
+  Optional[Array[String]]                                 $denygroups                      = undef,
+  Optional[Array[String]]                                 $denyusers                       = undef,
+  Boolean                                                 $gssapiauthentication            = $ssh::server::params::gssapiauthentication,
+  Boolean                                                 $hostbasedauthentication         = false,
+  Boolean                                                 $ignorerhosts                    = true,
+  Boolean                                                 $ignoreuserknownhosts            = true,
+  Boolean                                                 $kerberosauthentication          = false,
+  Optional[Array[String]]                                 $kex_algorithms                  = undef,
+  Optional[Variant[Simplib::Host, Array[Simplib::Host]]]  $listenaddress                   = undef,
```

Comment:
el7 allows host|addr:port

Reply:
Since no one has asked for this, we are going to leave it as is.

Reply:
See [simplib#189](https://github.com/simp/pupmod-simp-simplib/pull/189).